docs/HowTo/Get-Started/Run-Teku.md

```diff
@@ -98,6 +98,33 @@ Specify the beacon chain client using the
 [`--beacon-node-api-endpoint`](../../Reference/CLI/Subcommands/Validator-Client.md#beacon-node-api-endpoint)
 option.
 
+## Confirm Teku is running
+
+Use the [`/liveness`](https://consensys.github.io/teku/#operation/getTekuV1AdminLiveness) endpoint
+to check whether the node is up.
+
+The endpoint returns the status `200 OK` if the node is up or syncing.
+
+!!! example
+
+    === "curl HTTP request"
+
+        ```bash
+        curl -I -X GET "http://192.10.10.101:5051/teku/v1/admin/liveness"
+        ```
+
+    === "JSON result"
+
+        ```json
```

Comment:
Not sure this should be json markup - and shouldn't have `{ }` braces around it.  It's just text output.

Reply:
Yes, you're right. Remnants of my copy-paste :)